CHANGELOG.md

```diff
@@ -2,9 +2,33 @@
 
 ## Table of Contents
 
+- [v0.2.0-rc1](#v020-rc1)
 - [v0.1.0](#v010)
 - [v0.1.0-rc1](#v010-rc1)
 
+## v0.2.0-rc1
+
+### Notable changes since v0.1.0
+Introduction of Istio.
```

Comment:
Providers are now fetch resources and store them in their local storage. 
This is also true for ingress fetching. It is no longer happening on i2gw package and moved to be fetched at the provider level.

We onboarded two more providers; Istio and APISIX